sample-apps/sse/binary/enveloped/consumer.js

```diff
@@ -0,0 +1,25 @@
+(() => {
+    const key = 'API:KEY';
+    const channel = 'sample-app-sse';
+    const url = `https://realtime.ably.io/event-stream?channels=${channel}&v=1.1&key=${key}&delta=vcdiff`;
+    const eventSource = new EventSource(url);
+    const channelDecoder = new DeltaCodec.CheckedVcdiffDecoder();
+    
+    eventSource.onmessage = (event) => {
+        const message = JSON.parse(event.data);
+        let { id, data, extras } = message;

```

Comment:
If I'm to be really picky, `id` and `extras` should be `const`, not `let`.

i.e. I think your intent would be clearer communicated with two lines here:

```js
const { id, extras } = message;
let { data } = message;
```

Also applies to other examples in this PR.